scenarios/vault-integrated-storage/hashibox.yaml

```diff
@@ -135,6 +135,12 @@ provision:
 
       touch /shikari-bootstrapped
 
+copyToHost:
```

Comment:
For this scenario you should be copying the Vault CA file and not Consul and Nomad.

Reply:
On it!